core/modules/admin_bar/css/admin_bar.css

```
@@ -210,11 +210,18 @@
   margin: calc(-1px - var(--admin-bar-height)) 0 0 15em; /* LTR */
 }
 #admin-bar .dropdown li li.expandable.outside-right ul {
+  z-index: 1;
   margin-left: -15em; /* LTR */
+  box-shadow:
+    -1px 0 0 1px #777,
+    -8px 0px 24px 3px rgba(0, 0, 0, 0.6);
 }
 [dir="rtl"] #admin-bar .dropdown li li.expandable ul {
   margin-right: 15em;
   margin-left: 0;
+  box-shadow:
+    1px 0 0 1px #777,
+    8px 0px 24px 3px rgba(0, 0, 0, 0.6);
 }
 [dir="rtl"] #admin-bar .dropdown li li.expandable.outside-left ul {
   margin-right: -15em;
```

I think the RTL CSS might accidentally be in the wrong place:

```suggestion
}
[dir="rtl"] #admin-bar .dropdown li li.expandable.outside-left ul {
  margin-right: -15em;
  box-shadow:
    1px 0 0 1px #777,
    8px 0px 24px 3px rgba(0, 0, 0, 0.6);
```